models/src_amazon_ads.yml

```diff
@@ -159,12 +159,12 @@ sources:
           - name: state
             description: "{{ doc('state') }}"
 
-      - name: negative_keyword_history
+      - name: negative_keyword_history 
```

Comment:
What is a negative keyword?

Reply:
A negative keyword is a keyword for searches where you don't want your product to show up. The example I read was if you're selling plastic spoons, "wooden" could be a negative keyword so your product isn't shown for a wooden spoon search. I think this might be unique to amazon because I didn't see it in our other ad packages unless I missed it. Also I wasn't 100% sure how all the keywords would go together in the end, so I assume this will need adjusting once we see what actual data looks like. It seemed like they might all go to the keyword report, but I would need to verify.